docs/source/integration/saas/aws.md

```diff
@@ -77,35 +77,35 @@ Follow these steps to create an AWS role:
 ### Create AWS Custom Attributes in LDAP
 
 Now you need to add two new attributes into your Gluu LDAP. 
-Follow [these instructions](https://gluu.org/docs/ce/admin-guide/attribute/#add-the-attribute-to-ldap) to add new attributes in your LDAP server. 
+Follow [these instructions](https://gluu.org/docs/gluu-server/4.3/admin-guide/attribute/#add-custom-attributes-to-ldap) to add new attributes in your LDAP server. 
```

Comment:
the version here isn't right. It should be `4.4`

Reply:
Fixed